rpc_jobs/rpc_aio.yml

```diff
@@ -193,10 +193,11 @@
            {TRIGGER_USER_VARS}
          UPGRADE_FROM_REF: "{UPGRADE_FROM_REF}"
       - rpc_gating_params
-      - single_use_slave_params:
+      - instance_params:
          IMAGE: "{IMAGE}"
          FLAVOR: "{FLAVOR}"
          REGION: "{REGION}"
+      - jenkins_node_params
```

Comment:
Do we need to have the jenkins_node_params in each job? I don't think there are many use cases where we'd want to customize these parameters beyond the defaults other than in the long running slave job.